util/math.go

```diff
@@ -10,3 +17,19 @@ func Min(a ...int) int {
 	}
 	return min
 }
+
+// TryParseFloat takes an input string and validates that it is a valid finite
```

Comment:
Nit, but should read 'that is a numeric float64'.

Reply:
If you're talking about the comment it's accurate - the input can be any string, not just one that parses as a float64